### DISCUSSION TITLE
Generate thumbnails for old table column

### DISCUSSION THREAD

**Discussion by aasutossh** [pocketbase/pocketbase#4723] [Q&A]
I already have an old table in Pocketbase with image filed not set thumbnail column.
I set the thumbnail field now. I am not getting the resized thumbnail. Did I have to have it set before the files are uploaded or something?
I need to get the resized thumbnail for older images. Is it possible?

**Comment by ganigeorgiev** [comment]
I'm not sure I follow.

You don't have to do anything in addition as thumbnails are generated lazily on first access.

**Comment by aasutossh** [comment]
Turns out the request was getting cached by Postman. It works flawlessly.
Thank you :)
